src/app.js

```diff
@@ -132,6 +133,15 @@ var app = function(options) {
     });
   }
 
+  if (options.rootDocsRedirect) {
+    let link = options.docs.documenter.getDocumentationUrl();
+    DOCS_HTML = `<html><body><p>You're lost, here's the way out.</p>
+    <a href= "${link}" >Refer to the documentation</a></body></html>`;
+    app.get('/', function(req, res) {
+      res.status(404).send(DOCS_HTML);
+    });
+  }
+
```

Comment:
We will need a test for this new code, too.  Tests are in `test/app_test.js`.

Reply:
How to get 'docs' (instance of tc-lib-docs ) in app_test.js ?

Reply:
Probably the easiest way, since only one method gets used, is to make a fake object:

```javascript
let fakeDocs = {getDocumentationUrl: () => "https://fake.documentation/url"};
```

Reply:
okay, then I'll pass fakedocs in the app right? Please guide on how to write the testing function for getDocumentationUrl() in app.js.

Reply:
In the test script, add `docs: fakeDocs`, then write a test that fetches `/` and confirms that the HTTP status code is 404 and that it contains `<a href="https://fake.documentation/url">`.